pkg/cmd/server/start/master_args.go

```diff
@@ -84,6 +86,8 @@ func BindMasterArgs(args *MasterArgs, flags *pflag.FlagSet, prefix string) {
 
 	flags.StringVar(&args.EtcdDir, prefix+"etcd-dir", "openshift.local.etcd", "The etcd data directory.")
 
+	flags.StringSliceVar(&args.APIServerCAFiles, prefix+"certificate-authority", args.APIServerCAFiles, "Optional files containing signing authorities to use (in addition to the generated signer) to verify the API server's serving certificate.")
```

Comment:
not sure how I feel about this showing up here... if you want a config (which you need for namedCertificates), you probably want to run `start master --write-config` as a starting point, but then it feels silly to *also* have to run `create-master-certs` just to get additional certificate authorities in the kubeconfig files